.env.test

```diff
@@ -3,7 +3,8 @@ SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
 DEFAULT_INSTANCE_MYSQL_URL='mysql://root@127.0.0.1'
 DEFAULT_INSTANCE_MONGO_URL='mongodb://127.0.0.1'
-SWIFT_ENABLE=true
+# SWIFT_ENABLE=true  # Not used any longer
```

Comment:
What do you think of changing this comment to "Deprecated in favor of INSTANCE_STORAGE_TYPE setting"

Reply:
I forgot to remove this after testing.